src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -597,6 +726,13 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
     [asset, parsedPools],
   )
 
+  const confirmCopy = useMemo(() => {
```

Comment:
Will be able to be used for more things eventually i.e not enough balance, impossible to deposit because xyz etc